cookiecutter-django-app/README.rst

```diff
@@ -54,7 +54,7 @@ Enter the project and take a look around::
     $ ls
 
 Generate a virtualenv and generate requirements files with dependencies
-pinned to current versions (make sure you're using pip 9.0.2+ and Python 3.6)::
+pinned to current versions (make sure you're using pip 9.0.2+ and Python 3.8)::
```

Comment:
I don't know what minimum version of pip we should be recommending, if any. (Whole section probably deserves a review.)